components/form/Select.js

```diff
@@ -8,9 +8,12 @@ export default function Select({
   ...restProps
 }) {
   return (
-    <>
+    <div>
       {label && (
-        <label htmlFor={name} className={`${className} hidden md:block`}>
+        <label
+          htmlFor={name}
+          className={`dark:text-white ${className} hidden md:block`}
```

Comment:
Can you use `className` function https://github.com/EddieHubCommunity/BioDrop/blob/main/services/utils/classNames.js